### PR TITLE
WIP: Update to fix plugin workflow

### PR DESCRIPTION
**Pull request by larshesel**
- remove vmq_topic.erl as this already exists in vernemq itself
- introduce new vernemq_dev_topic file exposing the unword/1
- function. This will be the start of a documented, public API.

### CODE REVIEW

File: src/vernemq_dev_topic.erl [hunk 1]
```diff
@@ -0,0 +1,34 @@
+%% Copyright 2017 Erlio GmbH Basel Switzerland (http://erl.io)
```
Comment (codeadict) [inline src/vernemq_dev_topic.erl:1]: I'm curious why keep part of the functions here and the others in the VerneMQ core, any topic validation can be useful for plugin developers IMO and it doesn't hurt or expose harmful functionality.

Comment (larshesel) [reply src/vernemq_dev_topic.erl:1]: So far the only function I know that people are using is the `unword` function. We should add more functions, but if we start to give API guarantees it should be carefully thought out before doing so. We can't remove it afterwards.

So it's simply trying to be cautious.

Comment (codeadict) [reply src/vernemq_dev_topic.erl:1]: Thats very True too, I only use `unword` myself, not sure about others out there

Comment (larshesel) [reply src/vernemq_dev_topic.erl:1]: Guess we can't really know until someone complains or asks for something more... or we ourselves see a need for something.

Comment (codeadict) [reply src/vernemq_dev_topic.erl:1]: fair

**Review by codeadict — APPROVED** [review]

**Review by dergraf — APPROVED** [review]
LGTM!

**Pull request merged**